pkg/dependency/parser/java/pom/pom.go

```diff
@@ -303,7 +303,7 @@ func (d pomDependency) ToArtifact(opts analysisOptions) artifact {
 		Exclusions:   exclusions,
 		Locations:    locations,
 		Relationship: ftypes.RelationshipIndirect, // default
-		Test:         d.Scope == "test",
+		Test:         d.Scope == "test" || opts.testScope,
```

Comment:
Doesn't it mean it'll include the test even if testScope is false? Unlike the other PR, this PR still includes test dependencies for me

Reply:
hmm... that's weird.
Do you see any vulnerabilities for test dependencies (or test packages in `json` format)?

`debug` logs for test dependencies are normal for this PR, because Trivy currently analyzes all dependencies and excludes `Dev` dependencies later.

Reply:
It takes ages to scan and then fails with:
```
semaphore acquire: context deadline exceeded
```

Reply:
This is timeout error.
Trivy's runtime has increased because Trivy first checks all dependencies (including test ones), and only then excludes them from the report